app/views/users/show.html.erb

```diff
@@ -0,0 +1,90 @@
+<%= render "shared/search_navbar" %>
+<%= render "shared/profile_header" %>
+<%= render "shared/flash" %>
+<%= render "relationships/pending_friend_request" %>
+
+<div class="container">
+  <div class="col-md-4">
+    <div class="row">
+      <div class="col-md-12">
+        <h3 class="text-center">About<h3>
+        <p class="about-details">Name: <%= @user.first_name.capitalize %> <%= @user.last_name.capitalize %><br></p>
+        <p class= "about-details">College: <%= @profile.college %></br></p>
+        <p class= "about-details">Current Location: <%= @profile.current_location %></br></p>
+        <p class= "about-details">Hometown: <%= @profile.hometown %></br></p>
+      </div>
+    </div>
+    <div class="col-md-12">
+      <h3 class="text-center"> Photos<p>(<%= @user.photos.count %>)</p></h3>
+      <% @photos.each do |photo| %>
+        <div class="col-md-4 index-photo">
+          <%= link_to image_tag(photo.picture.url, class: "resize-index-photo"), user_photo_path(photo.user, photo) %>   
+        </div>
+      <% end %>
+    </div>
+    <div class="col-md-12">
+      <h3 class="text-center">Friends</h1>
+    </div>
+  </div>
+  <% if current_user && current_user.friends.include?(@user) %>
```

Comment:
Think you want || in place of && here.  Can be current user OR her friend right?